### PR TITLE
Fix errors in LeadPressureCalc algorithm

### PR DESCRIPTION
**Pull request by clayton-tom**
**Description of work.**
Fix errors found in LeadPressureCalc by Chris Ridley. Specifically, the EOS c constant was out by e3, and there was incorrect bracketing in the calculation.

Unit tests updated with externally calculated results.

**Report to:** Chris Ridley

**To test:**
Ensure unit tests pass

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by DannyHindson** [comment]
Looking at the changes.txt file that Chris attached in his email earlier there are one or two further differences in there so I think you should apply these also:

- a = 1.0582E-02
- c = -2.54374974975855E-02
- d = -2.7567006420938E-06
- k00 = 41.7253509951022
- k00p = 5.3944

The change to d is just some extra significant figures and arguably won't make much difference but we may as well match what he sent this morning exactly. Some of the others are different in the ~4th significant figure so could matter.

The original GitHub issue https://github.com/mantidproject/mantid/issues/30049 contains the values I've listed above. Do you know where the current values in the "master" code came from? Perhaps there were some further exchanges between you and Chris or maybe they came out of a paper?

**Comment by clayton-tom** [comment]
@DannyHindson the ones used came from the excel document Chris provided, will update to use these more precise values

**Review by DannyHindson — APPROVED** [review]
This looks good now thanks

File: Framework/PythonInterface/plugins/algorithms/LeadPressureCalc.py [hunk 1]
```diff
@@ -16,25 +16,25 @@
 # this calculation taken from the EOS equations in [1]
 def calculate_pressure(d_spacing, temp):
     v00 = 121.41813  # zero pressure unit cell volume at (0) (A^3)
```
Comment (martyngigg) [inline Framework/PythonInterface/plugins/algorithms/LeadPressureCalc.py:18]: I wonder if we could extract these constants as module level variables with capitalized names to indicate they are constants that might given them more meaning without the comments?

**Review by martyngigg — CHANGES_REQUESTED** [review]

File: Framework/PythonInterface/plugins/algorithms/LeadPressureCalc.py (hunk 1 above)
Comment (DannyHindson) [reply Framework/PythonInterface/plugins/algorithms/LeadPressureCalc.py:18]: Tom's off now until next Thurs and Chris Ridley has asked that we put this fix through quickly. Could Tom do the change to these variables as a separate issue to enable this fix to go into the nightly soon?

Comment (martyngigg) [reply Framework/PythonInterface/plugins/algorithms/LeadPressureCalc.py:18]: Yeah of course. Merging now.

**Review by martyngigg — APPROVED** [review]

**Pull request merged**